vite.config.js

```diff
@@ -19,6 +19,7 @@ export default defineConfig({
   resolve: {
     alias: {
       "@utils": path.resolve(__dirname, "./src/utils"),
+      "@images": path.resolve(__dirname, "./src/images"),
```

Comment:
not necessary you can remove this. images folder would be within `assets` anyways